build.gradle

```diff
@@ -20,9 +20,9 @@ intellij {
     version ideaVersion //for a full list of IntelliJ IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
     pluginName 'org.jboss.tools.intellij.openshift'
     if (ideaVersion.substring(3) != "2019.1") {
-        plugins 'java', 'terminal', 'JavaScriptDebugger', 'JavaScriptLanguage', 'CSS', 'yaml'
```

Comment:
I think this can be simplified as we now always requires the java plugin as we are now 2019.3+